package.json

```diff
@@ -18,6 +18,7 @@
     "start:e2e": "nx run autoscript-apps:serve",
     "start:embedded-login": "nx serve embedded-login",
     "start:central-login": "nx serve central-login",
+    "start:angular-todo": "nx run angular-todo:config && nx run-many --target=serve --projects=todo-api,angular-todo",
```

Comment:
Other than this change here, I think the rest of changes below to this file need to be reverted. These were changes recently introduced by @ryanbas21.

Reply:
Just fixed this with Ryan

Reply:
Should be in line with develop now